docs/components/radio.md

```diff
@@ -202,17 +198,19 @@ Any valid attribute will be added to the input. You can read more about `$attrs`
 </KRadio>
 ```
 
-- `description` - Anything passed in to this slot will replace the `description` prop text.
+:::warning NOTE
```

Comment:
suggestion: I don't think this needs to be a warning, more just like "info" or "tip"
```suggestion
:::tip NOTE
```